composer.json

```diff
@@ -50,6 +50,7 @@
         "liip/imagine-bundle": "1.3.0",
         "imagine/imagine": "0.6.2",
         "monolog/monolog": "1.22.0",
+        "ocramius/proxy-manager": "2.0.4",
```

Comment:
I think you should indeed just remove it, or fix it for PHP 5.6 too.
In `doctrine/migrations`, you have:
```json
"ocramius/proxy-manager": "^1.0|^2.0"
```